dd-sdk-android/src/main/kotlin/com/datadog/android/rum/tracking/UserActionTrackingStrategy.kt

```diff
@@ -5,8 +5,13 @@
  */
 package com.datadog.android.rum.tracking
 
+import com.datadog.android.rum.internal.instrumentation.gestures.GesturesTracker
+
 /**
  * A TrackingStrategy dedicated to user actions tracking.
  */
-interface UserActionTrackingStrategy :
-    TrackingStrategy
+internal interface UserActionTrackingStrategy :
+    TrackingStrategy {
+
+    val gesturesTracker: GesturesTracker
```

Comment:
Not sure this should be part of the interface. One could implement a UserActionTrackingStrategy without the need to have a Gesture tracker in it.